### PR TITLE
Add package rtree

### PR DESCRIPTION
**Pull request by StefanSalewski**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -14788,5 +14788,23 @@
...
+  },
+  {
+    "name": "rtree",
+    "url": "https://github.com/stefansalewski/RTree",
+    "method": "git",
+    "tags": [
+      "RTree",
+      "R-Tree",
+      "R*Tree",
+      "R*-Tree",
```
Comment (dom96) [inline packages.json]: Please remove these. These aren't good tags, they are just variations of the name. Tags should reflect some overarching categories like `library` or `wrapper` or `network` etc.

**Comment by StefanSalewski** [comment]
OK -- I clicked on something like "Resolve conversation".

But package seems to be not in database still. Have I to call nimble publish again, with shorter tags list?

No time to further investigations currently, sorry.

**Comment by narimiran** [comment]
> OK -- I clicked on something like "Resolve conversation".

That just hid the conversation, nothing else changed.

**Comment by StefanSalewski** [comment]
Seems to be not that easy, from

https://stackoverflow.com/questions/16748115/how-to-modify-github-pull-request

my feeling is that I have to learn a lot of git/github API. I will try to close this one and do a new nimble publish.

**Pull request closed without merging**